ci/upload-codecov.sh

```diff
@@ -16,21 +16,11 @@
 
 set -eu
 
-if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
-    echo "Skipping code coverage as it is disabled for pull requests."
-    exit 0
-fi
-
 if [ "${BUILD_TYPE:-Release}" != "Coverage" ]; then
     echo "Skipping code coverage as it is disabled for this build."
     exit 0
 fi
 
-if [ -z "${CODECOV_TOKEN:-}" ]; then
```

Comment:
Intentional deletion?

Reply:
Sorry, I was testing changes.  I did not think the PR was ready for review yet.